libraries/AP_HAL_Linux/RPIOUARTDriver.h

```diff
@@ -1,5 +1,7 @@
 #pragma once
 
+#include <AP_HAL/utility/RingBuffer.h>
```

Comment:
You are right, I had the wrong file on my screen, I thinks does entries can be removed. I will remove and test if there was a hidden reason on why I added here.